.storybook/cookie-policy.ts

```diff
@@ -0,0 +1,58 @@
+const COOKIE_ACCEPT_GDPR_NAME = 'GDPR';
+
+function setCookie(cookieName: string, value: string, expirationInDays = 365) {
+    const d = new Date();
+    d.setTime(d.getTime() + expirationInDays * 24 * 60 * 60 * 1000);
+    const expires = `expires=${d.toUTCString()}`;
+
+    document.cookie = `${cookieName}=${value};${expires};path=/`;
+}
+
+function getCookie(cookieName: string): string | null {
+    const name = cookieName + '=';
+    const decodedCookies = decodeURIComponent(document.cookie);
+    const cookies = decodedCookies.split(';');
+
+    for (const cookie of cookies) {
+        if (cookie.trim().startsWith(name.trim())) {
+            return cookie.trim().substring(name.length, cookie.length);
+        }
+    }
+
+    return null;
+}
```

Comment:
no need for a generic getCookie function, this could be as simple as:
```js
function hasGdprCookie(): bool {
    return document.cookie.split(/;\s/).includes('accept-GDPR=accepted');
}
```